common/changes/@microsoft/rush/user-danade-UpdateInit_2021-05-10-19-32.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Update \"rush init\" assets to use newer versions of Rush and PNPM",
```

Comment:
Given that options and entire files are being removed, we should provide more complete release notes.  (Particularly for a user who wants to upgrade Rush but is not ready to move to PNPM 6 yet.)